### ISSUE TITLE
No more keyboard lights

### ISSUE THREAD

**Issue by EKami** [seerge/g-helper#514]
## NOTE
GHelper 0.75.0.0 has broken my keyboard lights on my Zephyrus duo 16 2023 (GX650PY)

**Describe the bug**
Prior to updating from 0.69.0.0 my keyboard lights were working, with 0.75.0.0 it's not working anymore whatever preset I select.

**Clear scenario to Reproduce**
1. Open GHelper
2. Select any preset light (like rainbow)
3. Wait...

**Expected behavior**
Lights preset to work

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Laptop model: Zephyrus duo 16 2023 (GX650PY)

**Asus software**
- MyASUS installed
- GHelper

**Comment by EKami** [comment]
Update: Solved if I move the "brightness" one level below then one level above, but it's still a bug, the keyboard should not be set to brightness 0 on startup.

**Comment by seerge** [comment]
@EKami do you have "lower backlight brightness on battery" checked and are you on battery on that moment ? if so - it will turn off brightness when launched. That's ok

**Comment by EKami** [comment]
Where do I find this option? But no, I'm never on battery on that gaming laptop.

**Comment by seerge** [comment]
@EKami try this build, 

[GHelper.zip](https://github.com/seerge/g-helper/files/11634729/GHelper.zip)

**Comment by EKami** [comment]
Backlight does work now, with or without this built. The thing is that the update has reset my backlight to 0 but as soon as I've changed the backlight level the issue was fixed and it's now almost impossible for me to reproduce the issue.

I guess I can close this issue and consider it solved? I've let this open just because for first-timers of GHelper, if they get this bug it could be confusing to understand why it's happening.
